### PR TITLE
Xpring ESLint [1/X]

### PR DESCRIPTION
**Pull request by hbergren**
## High Level Overview of Change

Rename all the test files from `*-test.ts` to `*.test.ts` so that the test-file specific exceptions in `xpring-eslint` trigger for these.

### Context of Change

Pre-req for onboarding `xpring-js` to `xpring-eslint`.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

All the test files now end in `.test.ts`.

## Test Plan

`npm run test` still passes.

### CODE REVIEW

File: test/Xpring/xpring-client.test.ts [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-import RawTransactionStatus from '../../src/XRP/raw-transaction-status'
+
 import { XRPLNetwork } from '../../src'
+import XpringClient from '../../src/Xpring/xpring-client'
 import XpringError from '../../src/Xpring/xpring-error'
+import RawTransactionStatus from '../../src/XRP/raw-transaction-status'
+import TransactionStatus from '../../src/XRP/transaction-status'
+import FakeXRPPayIDClient from '../PayID/fakes/fake-xrp-pay-id-client'
+import FakeXRPClient from '../XRP/fakes/fake-xrp-client'
+import { testXRPTransaction } from '../XRP/fakes/fake-xrp-protobufs'
```
Comment (hbergren) [inline test/Xpring/xpring-client.test.ts:12]: This should really be in another PR, but it's really just moving around some import statements, and I'm lazy.

Comment (keefertaylor) [reply test/Xpring/xpring-client.test.ts:12]: Looks reasonable to me. If you're doing massive amounts of work to clean up a codebase you can be lazy.
